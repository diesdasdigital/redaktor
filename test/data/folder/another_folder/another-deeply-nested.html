{
  "markdown": "",
  "path": "folder/another_folder/another-deeply-nested",
  "updatedAt": "2020-08-28T12:38:38.009Z",
  "createdAt": "2020-08-28T12:38:38.009Z"
}