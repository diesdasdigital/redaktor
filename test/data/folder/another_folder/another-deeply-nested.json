{
  "default": {
    "required": {
      "title": "Another Deeply Nested",
      "pageType": "code-preview"
    },
    "fields": {},
    "components": []
  }
}
