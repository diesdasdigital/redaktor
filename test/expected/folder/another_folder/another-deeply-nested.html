<pre>{
  "data": {
    "en": {
      "required": {
        "title": "Another Deeply Nested",
        "pageType": "default"
      },
      "fields": {},
      "components": []
    }
  },
  "path": "folder/another_folder/another-deeply-nested"
}</pre>